tools/rum-models-generator/Sources/CodeGeneration/Generate/Transformers/Swift/JSONToSwiftTypeTransformer.swift

```diff
@@ -189,14 +189,14 @@ internal class JSONToSwiftTypeTransformer {
     /// - Otherwise, if **all** `oneOf` sub-schemas represent different `types`, enum cases will be named by
```

Comment:
Is this comment up to date with the change from oneOf to supporting oneOf + anyOf?

Reply:
no it's not, let me update that 👍